sync2jira/main.py

```diff
@@ -292,7 +301,8 @@ def handle_msg(msg, suffix, config):
     # GitHub '.issue.' is used for both PR and Issue
     # Check for that edge case
     if suffix == 'github.issue.comment':
-        if 'pull_request' in msg['msg']['issue'] and msg['msg']['action'] != 'deleted':
+        body = c.extract_message_body(msg)
```

Comment:
Rather than adding the call to `extract_message_body()` here, consider changing the function signature to receive the body directly instead of the message wrapper for it.  This will save the dict instantiation at line 289 at the possible cost of having to add a call to `extract_message_body()` in `listen()` at line 172 (but, since you would be removing the call from here, that would be a net-zero cost and an overall savings, at the cost of an extra line of code change).

Reply:
Done in a305ae3.